submissions/godsfavourOmoare/CSS/css/style.css

```diff
@@ -194,4 +191,12 @@ footer .t-condition p:first-child {
         flex: 50%;
     }
 
+}
+
+@media screen and (min-width: 1300px) {
```

Comment:
Well done @Bebek-hub, kindly use the standard screens sizes or viewports:
For  Extra small devices (phones, 600px and down), use @media only screen and (max-width: 600px) {...}

For Small devices (portrait tablets and large phones, 600px and up), use @media only screen and (min-width: 600px) {...}
For  Medium devices (landscape tablets, 768px and up) , use @media only screen and (min-width: 768px) {...}